os/arch/arm/src/imxrt/imxrt_config.h

```diff
@@ -238,6 +238,134 @@
 #define CONFIG_IMXRT_ENET_PRIO  NVIC_SYSH_PRIORITY_DEFAULT
 #endif
 
+#ifndef CONFIG_ETH0_PHY_KSZ8081
+#define CONFIG_ETH0_PHY_KSZ8081
+#endif
+
+#ifndef CONFIG_NET_ETH_PKTSIZE
+#define CONFIG_NET_ETH_PKTSIZE 590
+#endif
+
+#if !(defined(FSL_SDK_DISABLE_DRIVER_CLOCK_CONTROL))
+#define FSL_SDK_DISABLE_DRIVER_CLOCK_CONTROL 0
+#endif
+
+/*! @brief Construct a status code value from a group and code number. */
+#define MAKE_STATUS(group, code) ((((group)*100) + (code)))
+
+/*! @brief Construct the version number for drivers. */
+#define MAKE_VERSION(major, minor, bugfix) (((major) << 16) | ((minor) << 8) | (bugfix))
+#define FSL_SAI_DRIVER_VERSION (MAKE_VERSION(2, 1, 3)) /*!< Version 2.1.3 */
+/*@}*/
+
+#define ARRAY_SIZE(x) (sizeof(x) / sizeof((x)[0]))
+
+#ifndef   __STATIC_FORCEINLINE                 
+  #define __STATIC_FORCEINLINE                   __attribute__((always_inline)) static inline
```

Comment:
Correct the formatting

Reply:
Can you explain it in detail?
I cannot understand your point exactly..

Reply:
Remove the space before "#define" and use tab instead of space before __attribute__((always_inline)) static inline